frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -118,16 +118,23 @@ const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
       prop: 'statefulSets',
     },
   ];
-
-  if (knative) {
-    const knativeResource = knativeServingResources(namespace);
-    resources = [...resources, ...knativeResource];
-  }
+  let crdUtils = [];
+  resourceList.forEach((resource) => {
```

Comment:
Shouldn't this list of resources be shared with overview page?
Topology may need to augment this list, but there is a minimum set that drives which workloads and the data needed for the sidebar that can be shared.

Reply:
fixed in https://github.com/openshift/console/pull/2849/commits/509691db8dfc209cabe54be11251791b86922378